langchain4j-azure-ai-search/src/main/java/dev/langchain4j/store/embedding/azure/search/QueryType.java

```diff
@@ -0,0 +1,6 @@
+package dev.langchain4j.store.embedding.azure.search;
+
+public enum QueryType {
```

Comment:
minor: what about `SearchType` or `SearchMode`?

Reply:
I will refactor this to have something like `AzureAiSearchQueryType` as otherwise it's misleading with classes like `Query` that is used with `ContentRetriever`